libs/exavmlib/lib/Range.ex

```diff
@@ -89,6 +89,37 @@ defmodule Range do
             "got: #{inspect(first)}..#{inspect(last)}"
   end
 
+  @doc """
+  Returns the size of `range`.
+
+  ## Examples
+
+      iex> Range.size(1..10)
+      10
+      iex> Range.size(1..10//2)
+      5
+      iex> Range.size(1..10//3)
+      4
+      iex> Range.size(1..10//-1)
+      0
+
+      iex> Range.size(10..1//-1)
+      10
+      iex> Range.size(10..1//-2)
+      5
+      iex> Range.size(10..1//-3)
+      4
+      iex> Range.size(10..1//1)
+      0
+
+  """
+  @doc since: "1.12.0"
```

Comment:
We probably should not copy this line over, or change it to 0.7.0, so that it actually reflects the release version when it was added.